starknet_devnet/postman_wrapper.py

```diff
@@ -49,3 +50,4 @@ def load_mock_messaging_contract_in_l1(self, starknet, contract_address):
             self.mock_starknet_messaging_contract = EthContract(self.web3,address,w3_contract,abi,self.eth_account)
 
         self.postman = Postman(self.mock_starknet_messaging_contract,starknet)
+        self.l1_to_l2_message_filter = self.mock_starknet_messaging_contract.w3_contract.events.LogMessageToL2.createFilter(fromBlock="latest")
```

Comment:
Why wasn't this needed before?

Reply:
The way the eth filter works is it pops the results when the `get_new_entries()` function is used, and I found no other way of just retrieving the messages without them disappearing, so we need another filter to be able to return the messages to the output of `flush`

Reply:
Is it related to garbage collection? Like, if there is no reference to the object, it gets garbage-collected?

Reply:
I don't think so, I think it's just that `get_new_entries()`, which is also called in postman for it's filter property, simply "pops" the messages from the filter.

Reply:
ok, didn't realize it was used in starknet_wrapper